apps/rehabstod/src/pages/Patient/components/PatientHeader.tsx

```diff
@@ -14,7 +14,7 @@ export function PatientHeader({ patient }: { patient: Patient }) {
   }
 
   return (
-    <div className="bg-secondary-95 sticky top-0">
+    <div className="bg-secondary-95 sticky top-0 z-40">
```

Comment:
Decrease to 30. We can also make `TableHeaderCell` have `z-[11]` to be just above sticky rows at z-10.